eng/pipelines/official.yml

```diff
@@ -50,9 +52,6 @@ variables:
 - name: isScheduleTrigger
   value: ${{ eq(variables['Build.Reason'], 'Schedule') }}
 
-- name: isPRTrigger
```

Comment:
Isn't this still needed? Or is the intention to use `unofficial` for PR builds? Because I see `unofficial` doesn't have any triggers defined right now.